utils/card.css

```diff
@@ -0,0 +1,76 @@
+.card {
+    display: flex;
+    flex-direction: column;
+    max-width: 300px;
```

Comment:
Use rem

Reply:
This is the exact size of the card, thats why i gave it in px. i dont want it to be responsove or so.